index.ts

```diff
@@ -97,23 +85,19 @@ export const setupPlugin: RedshiftPlugin['setupPlugin'] = async (meta) => {
         throw new Error(`Unable to connect to Redshift cluster and create table with error: ${queryError.message}`)
     }
 
-    global.buffer = createBuffer({
-        limit: uploadMegabytes * 1024 * 1024,
-        timeoutSeconds: uploadSeconds,
-        onFlush: async (batch) => {
-            await insertBatchIntoRedshift(
-                { batch, batchId: Math.floor(Math.random() * 1000000), retriesPerformedSoFar: 0 },
-                meta
-            )
-        },
-    })
-
     global.eventsToIgnore = new Set(
         config.eventsToIgnore ? config.eventsToIgnore.split(',').map((event) => event.trim()) : null
     )
 }
 
-export async function onEvent(event: ProcessedPluginEvent, { global }: RedshiftMeta) {
+export async function exportEvents(events: ProcessedPluginEvent[], meta: RedshiftMeta) {
+    const eventsToExport = events.filter((event) => !meta.global.eventsToIgnore.has(event.event))
```

Comment:
Same comment as previous times - imo the blacklist pattern is bad practice.

Reply:
This has been this way for a long time with these plugins. Users rely on the pattern. 

Also note that it's probably the best approach. Usually people blacklist $autocapture and $feature_flag_called events but want to get everything else, including any new events, rather than having to update the config each time